README.md

```diff
@@ -1031,8 +1031,8 @@ Device Descriptor:
 
 ### permissions
 
-This command will run a binary on your device in order to set the OTP permissions, as these are not directly accessible from `picotool` on due to the default permissions settings required to fix  errata XXX on RP2350. 
-Because it runs a binary, the binary needs to be sign it if secure boot is enabled. The binary will print what it is doing over uart, which
+This command will run a binary on your device in order to set the OTP permissions, as these are not directly accessible from `picotool` on due to errata RP2350-E15 on RP2350 A2. 
+Because it runs a binary, the binary needs to be signed if secure boot is enabled. The binary will print what it is doing over UART, which
 can be configured using the UART Configuration arguments. You can define your OTP permissions in a json file, an example of which
```

Comment:
The bit about UART is actually wrong now, as the binary lights an LED instead, so maybe something like the following?

```suggestion
Because it runs a binary, the binary needs to be signed if secure boot is enabled. The binary will light an LED when running, which
can be configured using the `--led` argument. You can define your OTP permissions in a json file, an example of which
```

Reply:
Thanks!